engines/director/resource.cpp

```diff
@@ -736,12 +736,6 @@ bool ProjectorArchive::loadArchive(Common::SeekableReadStream *stream) {
 		tag = stream->readUint32BE();
 		size = bigEndian ? stream->readUint32BE() : stream->readUint32LE();
 
-		// endianness issue, swap size and continue
```

Comment:
There was a suite of files to test against: <https://github.com/59de44955ebd/unpacker.py/tree/main/test_files> Could you please check that every version in there still works?

Reply:
@rvanlaar Any update on this?

Reply:
@moralrecordings Any update on this? I just do not want to add a blind regression.

Reply:
Apologies, I ran out of time last week as it wasn't immediately obvious how to test these en-masse. I ended up making individual folders with each projector in them to disable the autodetection.
- `projector_d40_16bit.exe` finds and correctly loads the ProjectorArchive
- `projector_d40_mac_68k.data` does not, however this is the same behaviour as `master`
- `projector_d50_16bit.exe` finds and correctly loads the ProjectorArchive
- `projector_d50_32bit.exe` finds and correctly loads the ProjectorArchive
- `projector_d50_mac_68k.data` does not, however this is the same behaviour as `master`
- `projector_d50_mac_fat.data` does not, however this is the same behaviour as `master`
- `projector_d50_mac_ppc.data` does not, however this is the same behaviour as `master`
- All the rest are unsupported versions of Director.

Reply:
By "does not" I mean `ProjectorArchive::loadArchive` isn't called at all.